test/Migration.t.sol

```diff
@@ -0,0 +1,473 @@
+
+import { DogeGaySon } from "../src/GogeToken.sol";
+import { DogeGaySon1 } from "../src/TokenV1.sol";
+
+contract MigrationTesting is Utility, Test {
+
+    DogeGaySon  gogeToken_v2;
+    DogeGaySon1 gogeToken_v1;
+
+    address UNIV2_ROUTER = 0x10ED43C718714eb63d5aA57B78B54704E256024E; //bsc
+
+    address BnbPriceFeedOracle = 0x0567F2323251f0Aab15c8dFb1967E4e8A7D42aeE;
+
+    function setUp() public {
+        createActors();
+        setUpTokens();
+        
+        // Deploy v1
+        gogeToken_v1 = new DogeGaySon1();
+
+        uint BNB_DEPOSIT = 300 ether;
+        uint TOKEN_DEPOSIT = 22_345_616_917 ether;
+
+        IWETH(WBNB).deposit{value: BNB_DEPOSIT}();
+
+        // Approve TaxToken for UniswapV2Router.
+        IERC20(address(gogeToken_v1)).approve(
+            address(UNIV2_ROUTER), TOKEN_DEPOSIT
+        );
+
+        // Create liquidity pool.
+        // https://docs.uniswap.org/protocol/V2/reference/smart-contracts/router-02#addliquidityeth
+        // NOTE: ETH_DEPOSIT = The amount of ETH to add as liquidity if the token/WETH price is <= amountTokenDesired/msg.value (WETH depreciates).
+        IUniswapV2Router01(UNIV2_ROUTER).addLiquidityETH{value: 220 ether}(
+            address(gogeToken_v1),      // A pool token.
+            TOKEN_DEPOSIT,              // The amount of token to add as liquidity if the WETH/token price is <= msg.value/amountTokenDesired (token depreciates).
+            TOKEN_DEPOSIT,              // Bounds the extent to which the WETH/token price can go up before the transaction reverts. Must be <= amountTokenDesired.
+            220 ether,                  // Bounds the extent to which the token/WETH price can go up before the transaction reverts. Must be <= msg.value.
+            address(this),              // Recipient of the liquidity tokens.
+            block.timestamp + 300       // Unix timestamp after which the transaction will revert.
+        );
+
+        // enable trading for v1
+        gogeToken_v1.afterPreSale();
+        gogeToken_v1.setTradingIsEnabled(true, 0);
+
+        // Deploy v2
+        gogeToken_v2 = new DogeGaySon(
+            address(0x4959bCED128E6F056A6ef959D80Bd1fCB7ba7A4B), //0x4959bCED128E6F056A6ef959D80Bd1fCB7ba7A4B
+            address(0xe142E9FCbd9E29C4A65C4979348d76147190a05a),
+            100_000_000_000,
+            address(gogeToken_v1)
+        );
+    }
+
+    // Initial state test.
+    function test_migration_init_state() public {
+        assertEq(gogeToken_v1.marketingWallet(), address(0x4959bCED128E6F056A6ef959D80Bd1fCB7ba7A4B));
+        assertEq(gogeToken_v1.teamWallet(),      address(0xe142E9FCbd9E29C4A65C4979348d76147190a05a));
+        assertEq(gogeToken_v1.totalSupply(),     100_000_000_000 ether);
+        assertEq(gogeToken_v1.balanceOf(address(this)), gogeToken_v1.totalSupply() - gogeToken_v1.balanceOf(gogeToken_v1.uniswapV2Pair()));
+
+        assertTrue(gogeToken_v1.tradingIsEnabled());
```

Comment:
Either `assert` or `assertEq`, no more `assertTrue` lol.

Reply:
Ahhhh you're right lolll. Fixed